lib/client/Client.js

```diff
@@ -638,14 +680,22 @@ export default class Client {
   }
 
   _handleMailboxesSetResponse(response) {
-    return MailboxesSet.fromJSONObject(this, response[1]);
```

Comment:
Can you tell me please why do we deal with the response[1] and not response[0] for example?